firmware/hw_layer/adc/adc_inputs.cpp

```diff
@@ -156,11 +165,11 @@ static ADCConversionGroup adcgrpcfgSlow = {
 	.sqr3				= 0  // Conversion group sequence 1...6
 };
 
-AdcDevice slowAdc(&adcgrpcfgSlow);
+AdcDevice slowAdc(&adcgrpcfgSlow, slowAdcSampleBuf, sizeof(slowAdcSampleBuf));
```

Comment:
do you want to pass the size (in bytes) or the size in elements (`efi::size(slowAdcSampleBuf)`)?